### PR TITLE
Made the consultation form responsive for Samsung Fold Device

### LINKED ISSUE #3148: make consultation update page responsive
image

### PR DESCRIPTION
**Pull request by patelaryan7751**
Fixes #3148 

# Made the consultation form responsive for Samsung Fold Device
consultation form

### CODE REVIEW

**Review by developedBySJ — CHANGES_REQUESTED** [review]
Migrate to tailwind CSS buttons.

**Comment by patelaryan7751** [comment]
Hey, @developedBySJ I have migrated the buttons to tailwind.

**Comment by nihal467** [comment]
@developedBySJ after review,  provide the comment

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -1022,29 +1020,24 @@ export const ConsultationForm = (props: any) => {
                 m<sup>2</sup>
               </div>
               {/* End of Telemedicine fields */}
-              <div className="mt-4 flex justify-between">
-                <Button
-                  color="default"
-                  variant="contained"
+              <div className="mt-4 sm:flex grid sm:justify-between">
+                <button
+                  className="btn text-gray-900 bg-gray-200 border border-gray-300 focus:outline-none hover:bg-gray-400  font-medium  text-sm px-5 py-2.5 mt-2"
```
Comment (developedBySJ) [inline src/Components/Facility/ConsultationForm.tsx]: Use `btn-default` class for the secondary button
```suggestion
                  className="btn btn-default mt-2"
```

**Review by developedBySJ — CHANGES_REQUESTED** [review]

**Comment by patelaryan7751** [comment]
@developedBySJ I have done the changes.

**Review by Ashesh3 — APPROVED** [review]

**Comment by nihal467** [comment]
@patelaryan7751 can you merge the latest version of care and update the label to need testing

**Comment by patelaryan7751** [comment]
@nihal467 done.

**Comment by nihal467** [comment]
@patelaryan7751 
image

make it responsive

**Comment by Ritesh-Aggarwal** [comment]
Hey @patelaryan7751, why was this PR closed , Is issue #3148 completed by some other PR?

**Comment by Ritesh-Aggarwal** [comment]
Hey @patelaryan7751 , why was this PR closed, Is there any other PR fixing #3148 ?

**Comment by patelaryan7751** [comment]
Hey actually i tried to fix it the ui changed so i unassigned it
@Ritesh-Aggarwal

**Comment by patelaryan7751** [comment]
You can work on it it is not fixed yet

**Pull request closed without merging**